hoodie-hadoop-mr/pom.xml

```diff
@@ -126,10 +130,33 @@
               <dependencyReducedPomLocation>${project.build.directory}/dependency-reduced-pom.xml
               </dependencyReducedPomLocation>
               <minimizeJar>true</minimizeJar>
+              <relocations>
```

Comment:
This is needed for working with Hive 2.x

Reply:
okay shading. sure.